src/Shared/Recruit.Vacancies.Client/Infrastructure/Client/IProviderVacancyClient.cs

```diff
@@ -15,5 +16,6 @@ public interface IProviderVacancyClient
         Task UpdateDraftVacancyAsync(Vacancy vacancy, VacancyUser user);
         Task<ProviderEditVacancyInfo> GetProviderEditVacancyInfoAsync(long ukprn);
         Task<EmployerInfo> GetProviderEmployerVacancyDataAsync(long ukprn, string employerAccountId);
+        Task<IEnumerable<IApprenticeshipProgramme>> GetActiveApprenticeshipProgrammesAsync();
```

Comment:
I think this should be in `IRecruitVacancyClient` and Employer and Provider should point to it.